tests/mention/mention_helper_test.php

```diff
@@ -31,17 +37,10 @@ public function getDataSet()
     public function test_get_mentioned_users()
     {
         global $request;
-        // $notification_manager = $this->prophesize(manager::class)->reveal;
-        // $notif_type_class = $this->prophesize(mention::class)->reveal();
-        // $container = $this->prophesize(ContainerInterface::class);
-        // $container->get('notification_manager')->willReturn($notification_manager);
-        // $container = $container->reveal();
         $sample_text = 'cbwkf kfbwkef wkfwb @robdyrdek vghjgjdsrshc ftfkyugb buykyuf yiglyftydtrf gyg ygiy @RickyBahner vgjdrs jfytg jbhjv ygfjh @HackMurphy kdnkwe [url = #]@andream[/url] [url = http://www.google.com]google[/url]wifhr [url = http://localhost/phpbb/phpBB/memberlist.php?mode=viewprofile&u=13]LindsayM[/url] wewe [url = http://localhost/phpbb/phpBB/memberlist.php?mode=viewprofile&u=13] @LindsayM [/url][mention]admin[/mention][mention]admin[/mention] dwjpfw [mention]admin[/mention]';
-        $request = new phpbb_mock_request();
         $db = $this->getMock('\phpbb\db\driver\driver_interface');
-        // $notification_manager = $this->getMockBuilder('\phpbb\notification\manager')->disableOriginalConstructor()->setMethods(['get_item_type_class', 'load_object'])->getMock();
-        // $notification_manager = $this->getMock('\phpbb\notification\manager', array('get_item_type_class', 'load_object'));
-        $helper_container = new \phpbb\mention\helper\mention_helper($db, $notification_manager);
+        echo get_class($this->notification_manager);
```

Comment:
@paul999  I wrote this  so that you can check the test case as I have mailed you about the issue I am facing.